actioncable/lib/action_cable/server/configuration.rb

```diff
@@ -29,6 +29,22 @@ def channel_class_names
           Pathname.new(channel_path).basename.to_s.split('.').first.camelize
         end
       end
+
+      ADAPTER = ActionCable::StorageAdapter
+
+      # Returns constant of storage adapter specified in config/cable.yml
+      # If the adapter cannot be found, this will default to the Redis adapter
+      def storage_adapter
+        # "ActionCable::StorageAdapter::#{adapter.capitalize}"
+        adapter = config_opts['adapter']
+        adapter_const = "ActionCable::StorageAdapter::#{adapter.capitalize}"
+
+        if Object.const_defined?(adapter_const)
+          adapter_const.constantize
+        else
+          ADAPTER_BASE::Redis
+        end
+      end
```

Comment:
I think it'd be simpler and clearer to just raise if the constant is missing and then you can just implement the method like this:

``` ruby
def storage_adapter
  adapter = config_options.fetch('adapter') { 'redis' }
  "ActionCable::StorageAdapter::#{adapter.camelize}".constantize
end
```

That way it doesn't just magically use Redis if you make a typo in your config file